psi4/src/read_options.cc

```diff
@@ -1484,6 +1484,8 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_double("DFT_BS_RADIUS_ALPHA", 1.0);
         /*- DFT basis cutoff. -*/
         options.add_double("DFT_BASIS_TOLERANCE", 1.0E-12);
+        /*- grid weight cutoff. !expert -*/
+        options.add_double("DFT_WEIGHTS_TOLERANCE", 1.0E-14);
```

Comment:
I suppose the default should be the largest value that still passes full tests. I had a look at the `dft-smoke` test, the first change I see is on the "total points" line, where your test has ~2300 fewer grid points. The SAD energy is a full Hartree off...

Reply:
@PeterKraus before or after this change..?

Reply:
After this change, there are fewer points.

Reply:
Yeah I was wondering about the Hartree difference in the SAD energy. But yes, since the quadrature weights can be negative, which was not taken into account in the code, it's quite possible to get non-reasonable energies...